packages/dev/core/src/Rendering/fluidRenderer/fluidRendererGUI.ts

```diff
@@ -0,0 +1,453 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { KeyboardEventTypes } from "core/Events/keyboardEvents";
+import type { KeyboardInfo } from "core/Events/keyboardEvents";
+import { Color3 } from "core/Maths/math.color";
+import type { Observer } from "core/Misc/observable";
+import type { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+import { Tools } from "core/Misc/tools";
+
+import { FluidRenderingDebug } from "./fluidRenderingTargetRenderer";
+
+async function LoadLiLGUI(): Promise<void> {
+    return Tools.LoadScriptAsync("https://cdn.jsdelivr.net/npm/lil-gui@0.17.0/dist/lil-gui.umd.min.js");
+}
+
+declare let lil: any;
+
+namespace lil {

```

Comment:
Do we need the namespace here?
Need to be sure we are not exporting this namespace in the declarations. If possible to define this as a type and not namespace it would be better.

Reply:
I couldn't make the code work without doing a namespace, but I'm far from a Typescript expert...

Reply:
The class is going to be removed anyway.